core/arch/arm/kernel/thread.c

```diff
@@ -123,12 +123,18 @@ linkage uint32_t name[num_stacks] \
 
 #define GET_STACK(stack) ((vaddr_t)(stack) + STACK_SIZE(stack))
 
-DECLARE_STACK(stack_tmp, CFG_TEE_CORE_NB_CORE, STACK_TMP_SIZE, static);
 DECLARE_STACK(stack_abt, CFG_TEE_CORE_NB_CORE, STACK_ABT_SIZE, static);
+#ifdef CFG_NXP_SE05X_SVC
+DECLARE_STACK(stack_tmp, CFG_TEE_CORE_NB_CORE, 5 * STACK_TMP_SIZE, static);
+#ifndef CFG_WITH_PAGER
+DECLARE_STACK(stack_thread, CFG_NUM_THREADS, 2 * STACK_THREAD_SIZE, static);
```

Comment:
what do you mean? SE05X just needs larger stacks...why would it be cleaner setting them from its own configs?

Reply:
Imaging other platform specific features have also specific requirements on the stacks sizes. That would put a mess in this source file. It would be better that it is handle in a specific place, for example using configuration `CFG_STACK_TMP_SIZE`, CFG_STACK_THREAD_SIZE`, that consider other config settings to set the right value.

Reply:
you are right.